cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectorIntegrationTest.java

```diff
@@ -55,7 +55,7 @@
 
 @EnableKubernetesMockClient(crud = true)
 @ExtendWith(VertxExtension.class)
-public class KafkaConnectorIT {
+public class KafkaConnectorIntegrationTest {
```

Comment:
Why did you renamed all the tests? We keep integration tests with the IT suffix. I do not see any value in renaming them.